src/js/view/components/timeslider.js

```diff
@@ -207,6 +211,18 @@ define([
             }
 
             var timetipText;
+
+            // With touch events, we never will get the hover events on the cues that cause cues to be active.
+            // Therefore use the info we about the scroll position to detect if there is a nearby cue to be active.
+            if (evt.sourceEventType === 'touch') {
```

Comment:
Would using the UI util to add the appropriate event listeners in the chapters mixin work in lieu of doing this here?
https://github.com/jwplayer/jwplayer/blob/master/src/js/view/components/chapters.mixin.js#L62